Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -65,12 +65,12 @@ class OverviewViewController: UIViewController {
     var recentTransactions = [Transaction]()
     var refreshBestBlockAgeTimer: Timer?
     
+    var refreshControl: UIRefreshControl!
```

Comment:
Move this just above the declaration for `parentScrollView`. It is related to that view, related views should be declared close together.

Reply:
resolved in `698f737 `